src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -231,6 +237,7 @@ parseLocalDeclaration :: TokenParser Declaration
 parseLocalDeclaration = positioned (P.choice
                    [ parseTypeDeclaration
                    , parseValueDeclaration
+                   , parseBoundValueDeclaration
```

Comment:
maybe we need to use `parseBinderNoParens`? `parseBinderAtom` will mis-parse the case where there are constructors other than nullary one.

```hs
f X a b = ...
```